<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelsUI</class>
 <widget class="QWidget" name="ChannelsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>335</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Predefined sets:</string>
        </property>
        <property name="buddy">
         <cstring>predefinedlist</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="predefinedlist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemovePredefined">
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset resource="../../app.qrc">
          <normaloff>:/icons/edit-delete.png</normaloff>:/icons/edit-delete.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSavePredefined">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="../../app.qrc">
          <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionLoad_from_EEProm">
   <property name="icon">
    <iconset resource="../../app.qrc">
     <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Load from EEProm</string>
   </property>
  </action>
  <action name="actionSaveToEEProm">
   <property name="icon">
    <iconset resource="../../app.qrc">
     <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>SaveToEEProm</string>
   </property>
  </action>
  <action name="actionChannelCount">
   <property name="icon">
    <iconset resource="../../app.qrc">
     <normaloff>:/icons/chronometer.png</normaloff>:/icons/chronometer.png</iconset>
   </property>
   <property name="text">
    <string>ChannelCount</string>
   </property>
  </action>
  <action name="actionResend">
   <property name="icon">
    <iconset resource="../../app.qrc">
     <normaloff>:/icons/arrow-right.png</normaloff>:/icons/arrow-right.png</iconset>
   </property>
   <property name="text">
    <string>Resend</string>
   </property>
  </action>
  <action name="actionRefetch">
   <property name="icon">
    <iconset resource="../../app.qrc">
     <normaloff>:/icons/arrow-left.png</normaloff>:/icons/arrow-left.png</iconset>
   </property>
   <property name="text">
    <string>Refetch</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../app.qrc"/>
 </resources>
 <connections/>
</ui>
